dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -38,12 +38,14 @@ def create
     # Once this has been done, endpoint can use CanCan load_and_authorize_resource
     # rather than manually authorizing (above)
     return head :bad_request unless Section.valid_login_type? params[:login_type]
+    return head :bad_request unless Section.valid_participant_type? params[:participant_type]
```

Comment:
heads up that users may get some errors here around the time of the deploy.

because the section is created as an ajax request, anyone who loaded the homepage before the deploy, but then creates a section after, would get a error message due to this param being missing.

my recommendation would be to prevent these errors by providing a default value of "student" for the participant type here. if you want, you could remove the default value after a deploy or two.

to be fair, we're not very consistent as an eng team about trying to prevent these kinds of errors -- when there is a cluster of errors around a deploy, and it looks related to a feature that went out, we often ignore the errors and move on without investigating unless the errors recur. so if you want to go that route, that's ok too.